config/config.go

```diff
@@ -228,6 +229,7 @@ var defaultConf = Config{
 		CrossJoin:       true,
 		StatsLease:      "3s",
 		RunAutoAnalyze:  true,
+		StmtCountLimit:  1000,
```

Comment:
1000 may be too small.

Reply:
5k or 1w is better。

Reply:
This is not `insert into t values (1),(2),(3)...(1000)`, it's 

```
insert into t values (1)
insert into t values (2)
...
insert into t values (1000)
```

If the amplification ratio is 100, then
1k (one row) * 1000 (statement limit)  * 1000 (concurrency) * 100 (amplification) = 100G

Reply:
This is used to avoid extreme scenario. I think in most cases, 5k is big enough to not block the application and small enough to prevent tidb-server from OOM.